.gitattributes

```diff
@@ -12,24 +12,31 @@
 # Likewise, anything marked `export-ignore` will be excluded by `bin/export-plugin.sh`
 
 # Directories
-/.github             export-ignore
-/.wordpress-org      export-ignore
-/bin                 export-ignore
-/tests               export-ignore
+/.github                         export-ignore
+/.husky                          export-ignore
+/.wordpress-org                  export-ignore
+/bin                             export-ignore
+/tests                           export-ignore
 
 # Files
-/.editorconfig       export-ignore
-/.eslintrc           export-ignore
-/.gitattributes      export-ignore
-/.gitignore          export-ignore
-/.phpcs.xml.dist     export-ignore
-/.prettierrc         export-ignore
-/.nvmrc              export-ignore
-/.travis.yml         export-ignore
-/CHANGELOG.md        export-ignore
-/CODE_OF_CONDUCT.md  export-ignore
-/CONTRIBUTING.md     export-ignore
-/composer.json       export-ignore
-/composer.lock       export-ignore
-/package-lock.json   export-ignore
-/phpunit.xml.dist    export-ignore
+/.editorconfig                   export-ignore
+/.eslintrc                       export-ignore
+/.gitattributes                  export-ignore
+/.gitignore                      export-ignore
+/.phpcs.xml.dist                 export-ignore
+/.prettierrc                     export-ignore
+/.nvmrc                          export-ignore
+/.travis.yml                     export-ignore
+/babel.config.js                 export-ignore
+/CHANGELOG.md                    export-ignore
+/CODE_OF_CONDUCT.md              export-ignore
+/CONTRIBUTING.md                 export-ignore
+/composer.json                   export-ignore
+/composer.lock                   export-ignore
+/package.json                    export-ignore
+/package-lock.json               export-ignore
+/phpstan.neon                    export-ignore
+/phpunit.xml.dist                export-ignore
+/phpunit-integration.xml.dist    export-ignore
+/webpack.config.js               export-ignore
+/WINDOWS.md               		 export-ignore
```

Comment:
We seem to have an alignment issue here compared to the above lines?

Reply:
Tabs are being used for some reason.
<img width="688" alt="Screenshot 2022-02-24 at 00 30 41" src="https://user-images.githubusercontent.com/88371/155434063-1262aa9c-5cc2-442a-b881-25d0d52a2203.png">